sdk/core/perf/test/inc/azure/perf/test/delay_test.hpp

```diff
@@ -67,7 +67,9 @@ namespace Azure { namespace Perf { namespace Test {
      * @brief The test definition
      *
      */
-    void Run(Azure::Core::Context const&) override { std::this_thread::sleep_for(m_delay); }
```

Comment:
What changed to cause this? I would think that clang-format would be upset and one of these two forms.